solutions/security/get-started/content-connectors.md

```diff
@@ -0,0 +1,39 @@
+---
+applies_to:
+  stack: preview 9.1
+  serverless: preview
```

Comment:
```suggestion
  stack: 
  serverless:
```

because this is now an overview page and also mentions self-managed connectors that have been around for a long time, we have to keep this page-level tag very broad and just use it to imply that there is both stack and serverless relevant info